images/win/toolsets/toolset-2022.json

```diff
@@ -105,8 +105,16 @@
     ],
     "java": {
         "default": "8",
-        "versions": [
-            "8", "11"
+        "default_vendor": "Temurin-Hotspot",
+        "vendors": [
+            {
+                "name": "Temurin-Hotspot",
+                "versions": [ "8", "11" ]
+            },
+            {
+                "name": "Adopt",
+                "versions": [ "8", "11", "13" ]
```

Comment:
Why do we need to install Java 13 on Windows Server 2022? We don't have this version on this platform.

Reply:
removed